CHANGELOG.md

```diff
@@ -1,217 +1,241 @@
-| Configure dependabot to only open 5 pull requests and restrict to semver | **Update**  | [#2526][] [#2528][] |
-| Bumped http-proxy-middleware from 0.19.1 to 0.19.2                       | **Update**  | [#2532][]           |
-| Bumped apollo-link-retry from 2.2.15 to 2.2.16                           | **Update**  | [#2530][]           |
-| Bumped @apollo/react-hooks from 3.1.3 to 3.1.5                           | **Update**  | [#2529][]           |
-| Bumped lodash from 4.17.14 to 4.17.19 in /docker                         | **Update**  | [#2556][]           |
-| Bumped lodash from 4.17.15 to 4.17.19 in /pwa-devdocs                    | **Update**  | [#2555][]           |
-| Bumped elliptic from 6.5.2 to 6.5.3                                      | **Update**  | [#2593][]           |
-| Bumped elliptic from 6.5.2 to 6.5.3 in /pwa-devdocs                      | **Update**  | [#2596][]           |
-| Bumbed dot-prop to version 5.1.1 or later                                | **Update**  | [#2601][]           |
-| Bumped kramdown from 2.2.1 to 2.3.0 in /pwa-devdocs                      | **Update**  | [#2614][]           |
-| Removed dependabot version bump settings                                 | **Update**  | [#2646][]           |
-| Bumped bl from 3.0.0 to 3.0.1                                            | **Update**  | [#2675][]           |
-| Fixed a PageBuilder visibility bug affecting slider buttons and links    | **Bugfix**  | [#2722][]           |
+| Description                                                                   | Change type | PR        |
+| ----------------------------------------------------------------------------- | ----------- | --------- |
+| Removed the `venia-styleguide` package                                        | **Update**  | [#2706][] |
+| Update use of `Whitelist` to `Allowlist`                                      | **Update**  | [#2779][] |
+| Added `jsx-no-literals` linting rule                                          | **Update**  | [#2789][] |
+| Update PR template to add translation entry to checklist                      | **Update**  | [#2800][] |
+| Fixed Storybook in scaffolded projects                                        | **Bugfix**  | [#2708][] |
+| Fixed a Storybook bug related to `fetchLocaleData` in storybook config        | **Bugfix**  | [#2801][] |
+| Fix failing unit tests related to race conditions                             | **Bugfix**  | [#2880][] |
+| Fixed a Storybook bug related to the relative import of a local custom loader | **Bugfix**  | [#2912][] |
 
 ## Known issues
 
-- PWA Studio 8.0.0 is not fully compatible with Magento 2.3.6, which can prevent Users from using the Reset Password feature.
-- When switching stores as a logged in customer, the shopping cart is not reassigned to the new store.
+- A GraphQL issue prevents users from adding a configurable product on non-default store views.
```

Comment:
So oddly enough I was able to add configurable products on non-default store views using `2.4.2-develop` backend. I'm not certain when that will be released though.

Reply:
Looking at the RCA done by Honey Badgers, it seems they narrowed it down to an issue with MSI, which isn't part of the core monorepo.

Reply:
In that case, maybe we don't need to include this line. Or maybe update it to say "If using MSI there is a known issue when adding configurable items to a cart on non-default store view"? Provide link to MSI issue (if public)